semgrep/semgrep/core_runner.py

```diff
@@ -336,7 +301,6 @@ def _run_rules_direct_to_semgrep_core(
                         cmd = [SEMGREP_PATH] + [
                             "-lang",
                             language.value,
-                            "-fast",
```

Comment:
`-fast` skips files that do not contain tokens in the pattern. Removing it when optimizations are set to none so parsing tests do not skip parsing files.

Reply:
Nice, makes a lot of sense